### PR TITLE
Add a link to the pr commit in the apks comment

### PR DESCRIPTION
**Pull request by suricactus**
Let's see how it looks

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for https://github.com/${ github.repository }/pull/${ github.event.pull_request.number }/commits/${ github.sha }:
  - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-66afa1e4514443e215faac2920f7fa8ac8e4a046-arm64_v8a.apk)

Other architectures: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-66afa1e4514443e215faac2920f7fa8ac8e4a046-armv7.apk), [x86_64](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-66afa1e4514443e215faac2920f7fa8ac8e4a046-x86_64.apk), [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-66afa1e4514443e215faac2920f7fa8ac8e4a046-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for https://github.com/opengisch/QField/pull/1435/commits/10d03b6e261abd8d983228f42098da323b98c4e2:
  - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-10d03b6e261abd8d983228f42098da323b98c4e2-arm64_v8a.apk)

Other architectures: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-10d03b6e261abd8d983228f42098da323b98c4e2-armv7.apk), [x86_64](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-10d03b6e261abd8d983228f42098da323b98c4e2-x86_64.apk), [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-10d03b6e261abd8d983228f42098da323b98c4e2-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for https://github.com/opengisch/QField/pull/1435/commits/f6de5524c8b5f535657f8eab3503853625ffd8bc:
  - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-f6de5524c8b5f535657f8eab3503853625ffd8bc-arm64_v8a.apk)

Other architectures: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-f6de5524c8b5f535657f8eab3503853625ffd8bc-armv7.apk), [x86_64](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-f6de5524c8b5f535657f8eab3503853625ffd8bc-x86_64.apk), [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-f6de5524c8b5f535657f8eab3503853625ffd8bc-x86.apk)

File: .github/workflows/continuous_integration.yml [hunk 1]
```diff
@@ -122,7 +122,7 @@ jobs:
         with:
           GITHUB_TOKEN: ${{ secrets.FAIRY_TOKEN }}
           message: |
-            🎉 Ta-daaa, freshly created APKs are available:
+            🎉 Ta-daaa, freshly created APKs are available for https://github.com/${{ github.repository }}/pull/${{ github.event.pull_request.number }}/commits/${{ github.sha }}:
```
Comment (m-kuhn) [inline .github/workflows/continuous_integration.yml:125]:
```suggestion
            🎉 Ta-daaa, freshly created APKs are available for ${{ github.sha }}:
```

I think that should do the same and be shorter to read (only showing the first 7 chars from the sha would be even better)

Comment (suricactus) [reply .github/workflows/continuous_integration.yml:125]: Let's try it.

Comment (suricactus) [reply .github/workflows/continuous_integration.yml]: btw, it is already showing the first 7 chracters in the comments?

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for befe2e837886901082bd13c24d9a14614d627afe:
  - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-befe2e837886901082bd13c24d9a14614d627afe-arm64_v8a.apk)

Other architectures: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-befe2e837886901082bd13c24d9a14614d627afe-armv7.apk), [x86_64](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-befe2e837886901082bd13c24d9a14614d627afe-x86_64.apk), [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-befe2e837886901082bd13c24d9a14614d627afe-x86.apk)

File: .github/workflows/continuous_integration.yml (hunk 1 above)
Comment (suricactus) [reply .github/workflows/continuous_integration.yml]: @m-kuhn okay, works, ready to merge :)

**Review by m-kuhn — APPROVED** [review]

**Comment by m-kuhn** [comment]
Silly me, it didn't change anything :laughing:

**Comment by suricactus** [comment]
But it's wrong, look the sha in the comment is actually the base sha :(

**Comment by m-kuhn** [comment]
Hover over it, it's a merge of commit into master

**Comment by suricactus** [comment]
Hm, ok. Silly me.

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for e942e0994d1fad827e752b88bc33929ef2128fa9:
  - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-e942e0994d1fad827e752b88bc33929ef2128fa9-arm64_v8a.apk)

Other architectures: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-e942e0994d1fad827e752b88bc33929ef2128fa9-armv7.apk), [x86_64](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-e942e0994d1fad827e752b88bc33929ef2128fa9-x86_64.apk), [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-e942e0994d1fad827e752b88bc33929ef2128fa9-x86.apk)

**Comment by m-kuhn** [comment]
See https://github.com/peter-evans/commit-comment

> or for pull_request events `github.event.pull_request.head.sha`

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for https://github.com/opengisch/QField/pull/1435/commits/c88ef60578bced093802d13dd3f17e99a43c626c:
  - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-c22aa9f42a0474c9ea42627b930466ff78cb8e0a-arm64_v8a.apk)

Other architectures: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-c22aa9f42a0474c9ea42627b930466ff78cb8e0a-armv7.apk), [x86_64](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-c22aa9f42a0474c9ea42627b930466ff78cb8e0a-x86_64.apk), [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-c22aa9f42a0474c9ea42627b930466ff78cb8e0a-x86.apk)

**Comment by suricactus** [comment]
Bingo!

File: .github/workflows/continuous_integration.yml [hunk 2]
```diff
@@ -122,7 +122,7 @@ jobs:
         with:
           GITHUB_TOKEN: ${{ secrets.FAIRY_TOKEN }}
           message: |
-            🎉 Ta-daaa, freshly created APKs are available:
+            🎉 Ta-daaa, freshly created APKs are available for https://github.com/${{ github.repository }}/pull/${{ github.event.pull_request.number }}/commits/${{ github.event.pull_request.head.sha }}:
```
Comment (suricactus) [inline .github/workflows/continuous_integration.yml:125]:
```suggestion
            🎉 Ta-daaa, freshly created APKs are available for ${{ github.event.pull_request.head.sha }}:
```

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 17c56b6be2aa00f29769329f2c56948ebd1c2d6c:
  - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-64e0e0ebaed83e6cbdd2b4e765d4ea836bbba8b0-arm64_v8a.apk)

Other architectures: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-64e0e0ebaed83e6cbdd2b4e765d4ea836bbba8b0-armv7.apk), [x86_64](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-64e0e0ebaed83e6cbdd2b4e765d4ea836bbba8b0-x86_64.apk), [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1435-64e0e0ebaed83e6cbdd2b4e765d4ea836bbba8b0-x86.apk)

**Pull request merged**